.github/workflows/test.yml

```diff
@@ -44,6 +45,7 @@ jobs:
         python -m vulture art/ otherfile/ setup.py art_profile.py --min-confidence 65 --exclude=__init__.py --sort-by-size
         python -m bandit -r art -s B311
         python -m pydocstyle --match='(?!test).*\.py' -v
+      if: matrix.python-version == 3.7
     - name: Codecov
```

Comment:
Please add a condition to this section to call `codecov` only in (`ubuntu` && `python 3.7`)